packages/react/src/lib/positioner/Popper.tsx

```diff
@@ -174,6 +175,8 @@ const Popper: React.FunctionComponent<PopperProps> = props => {
           placement: computedPlacement,
           scheduleUpdate,
         })
+      : children && rest && Object.keys(rest).length > 0
+      ? React.cloneElement(React.Children.only(children as React.ReactElement<any>), rest)
```

Comment:
Looks complicated, is a there any to simplify it?

Reply:
I can extract the condition to a separate const or function, but I guess is more clear to see it inlined, instead of scrolling back.. As for simplifying the condition, I don't see how. Do you have some suggestion? Btw the same condition is in the Ref component already.